backend/driver.pm

```diff
@@ -28,14 +28,9 @@ use IO::Select;
 use POSIX '_exit';
 require IPC::System::Simple;
 use autodie ':all';
+use bmwqemu qw(fctwarn logdie fctdbg);
 use myjsonrpc;
 
-# TODO: move the whole printing out of bmwqemu
```

Comment:
You did not really solve that TODO but IMHO made it worse. Didn't you suggest to use the logging modules directly where used?